src/rf/apps/core/decorators.py

```diff
@@ -0,0 +1,160 @@
+# -*- coding: utf-8 -*-
+from __future__ import print_function
+from __future__ import unicode_literals
+from __future__ import division
+
+import json
+from functools import wraps
+
+from django.http import HttpResponse, QueryDict
+from django.http.response import Http404
+
+from apps.core.exceptions import (BadRequest, Forbidden, InvalidApiCredentials,
+                                  MethodNotAllowed, Unauthorized)
+from apps.core.models import User
+
+
```

Comment:
Besides being decorator-based, I don't think there's much of an overlap. The routing logic that `django-tinsel` performs with decorators exists in the view functions here.